include/swift/AST/Attr.h

```diff
@@ -2183,6 +2204,19 @@ class TypeSequenceAttr : public DeclAttribute {
   }
 };
 
+// /// The @_noMetadata attribute, which marks a generic param as not providing metadata
```

Comment:
Is this intentional commented out like this or can this be remove in a follow-up PR?

Reply:
Good catch. This is a leftover from porting the attribute to swift-syntax. I'll remove it in a follow up.